pkg/connector/user.go

```diff
@@ -50,6 +52,17 @@ func (o *userResourceType) List(
 	resourceID *v2.ResourceId,
 	token *pagination.Token,
 ) ([]*v2.Resource, string, annotations.Annotations, error) {
+	if o.connector != nil && o.connector.awsConfig != nil && o.connector.awsConfig.Enabled {
```

Comment:
How could o.connector be nil at this point? Isn't it always set in userBuilder?

Reply:
theres a `ciamUserBuilder` which isn't taking connector

Reply:
I updated it to take connector